yaksh/templates/yaksh/post_comments.html

```diff
@@ -0,0 +1,72 @@
+{% extends "user.html" %}
+{% load static %}
+{% load humanize %}
+{% block title %}
+    {{post.title}}
+{% endblock title %}
+
+{% block content %}
+    <div class="container">
+        <a class="btn btn-primary" href="{% url "yaksh:course_forum" post.course.id %}">Back to Posts</a>
```

Comment:
Change the URL format from `{% url "yaksh:course_forum" post.course.id %}` to `{% url 'yaksh:course_forum' post.course.id %}`. Change this format wherever necessary.